.eslintrc.json

```diff
@@ -21,5 +21,5 @@
     "no-shadow": "off",
     "react/prop-types": "off"
   },
-  "ignorePatterns": ["dist/", "build/", "docs/"]
+  "ignorePatterns": ["dist/", "build/", "docs/","./src/Logic/calculate.js"]
 }
```

Comment:
- [ ] You did well removing most eslint disabled comments but a general rule is for you not to make custom changes to linter config files. I am sorry but I have to make the hard decision to invalidate this pull request. Kindly undo the changes and submit them again. 👍

Reply:
dist,docs folder and the build folder should be ignored but not `"./src/Logic/calculate.js"`